jobs/snort/templates/bin/pre-start

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+cat << 'CROND' > /etc/cron.hourly/snort-to-prometheus
+#!/bin/sh
+
+tempfile=$(mktemp)
+while IFS= read -r line; do
+cat <<DETECTION >> ${tempfile}
+snort_alert_detected {message=${line}} $(date +%s)
+DETECTION
+echo "${line}"
+done < /var/vcap/jobs/node_exporter/config/snort.msg
+
+mv ${tempfile} /var/vcap/jobs/node_exporter/config/$(date +"%FT%H%M%S%S")-snort-alert-detection.prom
+CROND
+
+chmod +x /etc/cron.hourly/snort-to-prometheus
```

Comment:
Hourly seems a bit long if there is a real issue detected. Could we do it every 5 minutes?